xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/AccountSet.java

```diff
@@ -265,4 +265,13 @@ public int getValue() {
       return value;
     }
   }
+
+  /**
+   * Sets an alternate account that is allowed to mint NFTokens on this
+   * account's behalf using NFTokenMint's `Issuer` field.
+   *
+   * @return An {@link Optional} field MintAccount of type {@link Address}.
+   */
+  @JsonProperty("MintAccount")
+  Optional<Address> mintAccount();
```

Comment:
not sure where this lives, but you will also need to add the ASF_AUTHORIZED_MINTER account set flag

Reply:
@mukulljangid The `ASF_AUTHORIZED_MINTER` flag should go in `AccountSetFlags` in this class.

Also, can you move this field declaration up with the other fields?